jerry-core/jrt/jrt.h

```diff
@@ -184,7 +181,7 @@ template<typename... values> extern void jerry_ref_unused_variables (const value
 /**
```

Comment:
I don't think that the argument name should be removed, as there is no implementation, corresponding to the function prototype.